docs/_quarto.yml

```
@@ -35,18 +27,25 @@ website:
           - custom_handler.md
       - text: "Changelog"
         file: changelog.md
-      - text: "Vetiver main site"
-        href: https://vetiver.rstudio.com
+      - text: "Learn more"
+        menu:
+          - text: "vetiver.rstudio.com"
+            href: https://vetiver.rstudio.com
+          - text: "R package documentation"
+            href: https://rstudio.github.io/vetiver-r/reference/
```

```suggestion
          - text: "vetiver.rstudio.com"
            href: https://vetiver.rstudio.com
            target: _blank
          - text: "R package documentation"
            href: https://rstudio.github.io/vetiver-r/reference/
            target: _blank
```

Since these take you off this site (I need to add this to the R docs)

---

Added to the R docs in rstudio/vetiver-r@dbeee3dcf1ef79c945525a167ba625566fea1127